VMBackup/main/handle.py

```diff
@@ -582,7 +621,8 @@ def daemon():
         backup_logger.commit_to_local()
         if(eventlogger is not None): 
             eventlogger.dispose()
-
+    if monitor_process is not None:
+        monitor_process.terminate()
```

Comment:
line 588 uses os.kill while we use process.terminate here. Why is the difference?